test/e2e/util.go

```diff
@@ -299,3 +364,39 @@ func readTestData(namespace, pvc string) ([]byte, error) {
 
 	return stdout, nil
 }
+
+func seekImageAndSnapNames(namespace, pool string) ([]string, error) {
```

Comment:
```suggestion
func getImageAndSnapNames(namespace, pool string) ([]string, error) {
```

`get` is more appropriate word.